examples/execd/Cargo.toml

```
@@ -0,0 +1,12 @@
+[package]
+name = "examples-execd"
+version = "0.1.0"
+edition = "2021"
+rust-version = "1.56"
```

Now that #318 has merged, this needs to be 1.58 to match the other examples:

```suggestion
rust-version = "1.58"
```

---

Fixed in 3aef1f6